packages/falso/src/tests/food.spec.ts

```diff
@@ -1,27 +1,47 @@
-import { randFood } from '../lib/food';
+import { FoodOrigin, randFood } from '../lib/food';
 import { data } from '../lib/food.json';
-import { seed } from '../lib/random';
+import * as coreFunctions from '../lib/core/core';
 
 describe('food', () => {
-  beforeEach(() => {
-    // Note: If more origins are added, some tests will fail
-    seed('🍔🌯🥗🍣🧁🦐🍽');
+  afterAll(() => {
+    jest.resetAllMocks();
   });
 
   describe('origin is passed', () => {
+    let origin: FoodOrigin;
+
+    beforeEach(() => {
+      origin = 'italy';
+    });
+
     it('should return a random dish from selected origin', () => {
-      const result = randFood({ origin: 'italy' });
+      const result = randFood({ origin: origin });
 
-      expect(data['italy']).toContain(result);
-      expect(result).toEqual('Il tartufo');
+      expect(data[origin]).toContain(result);
     });
   });
 
   describe('origin is not passed', () => {
+    let getRandomInRangeSpy: jest.SpyInstance;
+
+    beforeEach(() => {
+      getRandomInRangeSpy = jest
+        .spyOn(coreFunctions, 'getRandomInRange')
+        .mockImplementation(() => 1);
+    });
+
+    afterEach(() => {
+      jest.clearAllMocks();
+    });
+
     it('should return a random dish from a random origin', () => {
+      const originIndex = 1;
```

Comment:
If this shares the same value returned from the mocked implementation please move the const to a higher scope and share it.